docs/tutorial.rst

```diff
@@ -203,10 +206,11 @@ and making sure that no recombinations occurred in that region.
    the RecombinationMap class; setting ``discrete=True`` will more closely
    match the recombination model of SLiM.
 
+.. _sec_tutorial_simplification:
 
-**************
+++++++++++++++
 Simplification
```

Comment:
This and the next section ("adding mutations") aren't really part of "combining forward and coalescent simulations" - they're things you'd want to do even without any coalescent simulations at all. I don't think we need the "Combining..." top-level section at all?

Reply:
Ah, that was because the entire 3-part section has its own introduction. Previously that was the introduction used for the entire tutorial, but it didn't really fit because it was only relevant to the "recapitation -> simplification -> mutation" part. I guess a simple way to change this is to change the section heading to "recapitation, simplification, and mutation". But perhaps there's a better way?

Reply:
Right. This is a symptom of the docs being just cobbled together, and needing an overhaul. Those three things are together because they're part of the expected workflow. Maybe a heading titled "The standard workflow"? Or, "Common preprocessing steps"? Or, "The first things to do with your tree sequence"? Or, maybe just "Recapitation, simplification, and mutation", as you say; we'll rearrange later.

Reply:
I changed it to "Recapitation, simplification, and mutation", and added a bit of intro to explain that this is a common workflow.

I have overhauled some of the other sections a bit, and I think that makes them a tiny bit less of a hotchpotch. But I don't want to do much more (especially if we are going to move this over to a jupyter-book anyway).